draft-dekater-scion-controlplane.md

```diff
@@ -1316,10 +1321,10 @@ The path discovery mechanism balances the number of discovered paths and the tim
 The resource costs for path discovery are communication overhead, processing and storage. Communication is transmitting the PCBs and occasionally obtaining the required PKI material. Processing cost is validating the signatures of the AS entries, signing new AS entries, and, to a lesser extent, evaluating the beaconing policies. Storage is both the temporary storage of PCBs before the next propagation interval, and the storage of complete discovered path segments.
 All of these depend on the the number and length of the discovered path segments, that is, on the total number of AS entries of the discovered path segments.
 
-Interesting metrics for scalability and speed of path discovery are the time until all discoverable path segments have been discovered after a "cold boot", and the time until new link is usable.
-Generally, the time until a specific PCB is built depends on its length and the propagation interval.
-At each AS, the PCB will be processed and propagated at the subsequent propagation event. As propagation events are not synchronized between different ASes, a PCB arrives at a random point in time during the interval and is buffered before potentially being propagated.
-With a propagation interval T at each AS, the mean time until the PCB is propagated in one AS therefore is T / 2 and the mean total time for the propagation steps of a PCB of length L is L * T / 2 (with a variance of L * T^2 / 12).
+Interesting metrics for scalability and speed of path discovery are the time until all discoverable path segments have been discovered after a "cold start", and the time until new link is usable.
+Generally, the time until a specific PCB is built depends on its length, the propagation interval, wether on-path ASes use "fast recovery".
```

Comment:
wether -> whether

The fast recovery mechanism only significantly affects the path discovery time for (the first) down segments.
In core beaconing, it's mostly irrelevant for establishing connectivity, as it is disabled as soon as one PCB was propagated over an interface. Because origination/propagation starts from everywhere, this does not create any meaningful connectivity.

If you want to mention it at all, I would describe it specifically for the down beaconing case, at the end of the paragraph starting with "On a cold start of the network ...". Something like:
"When all ASes start propagation interval just after they've received the first PCBs from any of their providers (see 'fast recovery'), the construction of a first path to connect each AS to the ISD core is accelerated.".

Reply:
Thanks, I added your wording there.